### DISCUSSION TITLE
access to nested exception properties?

### DISCUSSION THREAD

**Discussion by Kosta-Github** [prisma/prisma#10727] [Q&A] [answered]
I have a trigger on one of my tables (MySQL), which signals an exception via:
```sql
SIGNAL SQLSTATE '45000' SET MESSAGE_TEXT = 'my_error_code'
```

When running a DB operation, which triggers this exception, I am receiving this kind of exception from prisma:
```text
PrismaClientUnknownRequestError: 
Invalid `prisma.my_table.create()` invocation in my_file.ts:132:50
...

Error occurred during query execution:
ConnectorError(ConnectorError { user_facing_error: None, kind: QueryError(Server(ServerError {
  code: 1644, message: "my_error_code", state: "45000" })) })
    at ...
  clientVersion: '3.6.0'
}
```

Is there a way to get access to this nested `message` property somehow? That is without parsing/regex'ing the `err.message` text...

**Comment by matthewmueller** [comment, marked as answer]
Hey @Kosta-Github, that's an interesting idea! I didn't know you could set custom errors like this.

Would you mind opening feature request? https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=

A reproduction would also be appreciated.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
